chapters.md

```diff
@@ -3,16 +3,52 @@ layout: page
 title: Join a Chapter
 permalink: /chapters/
 ---
+
 {% assign activity_levels = "active,inactive" | split: ','%}
+
 {% for level in activity_levels %}
-  <h2>{% if level == "inactive" %}Inactive and former chapters{% else %}{{level | capitalize}} chapters{% endif %}</h2>
-  {% assign chapters = site.data.chapters.chapters | where:"activity_level",level | sort: "text" %}
-  {% for chapter in chapters %}
-  <ul class="list-style-none marg-b-3 pad-l-3">
-    <b>{% if chapter.url %}<a href="{{chapter.url}}">{{chapter.text}}</a>{% else if%}{{chapter.text}}{% endif %}</b>
-    {% if chapter.twitter %} · <a href="https://twitter.com/{{chapter.twitter}}">Twitter</a>{% endif %}
-    {% if chapter.meetup %} · <a href="https://meetup.com/{{chapter.meetup}}">Meetup</a>{% endif %}
-    {% if chapter.facebook %} · <a href="{{chapter.facebook}}">Facebook</a>{% endif %}
-  </ul>
+  <section>
+    {% if level == "inactive" %}
+      <h2>Inactive and former chapters</h2>
+    {% else %}
+      <h2>Active chapters</h2>
+    {% endif %}
+
+    {% assign chapters = site.data.chapters.chapters | where:"activity_level",level | sort: "text" %}
+
+    <ul class="chapter-list">
+      {% for chapter in chapters %}
+        <li>
+          <span>{{chapter.icon}}</span>
+
+          {% if chapter.url %}
+            <a href="{{chapter.url}}" class="name">{{chapter.text}}</a>
+          {% else if%}
+            <span class="name">{{chapter.text}}</span>
+          {% endif %}
+
+          <!-- Icons on this page are CC0 https://simpleicons.org/ -->
+
+          {% if chapter.twitter %}
+            <a href="https://twitter.com/{{chapter.twitter}}">
+              <svg class="simple-icon twitter" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>X</title><path d="M14.234 10.162 22.977 0h-2.072l-7.591 8.824L7.251 0H.258l9.168 13.343L.258 24H2.33l8.016-9.318L16.749 24h6.993zm-2.837 3.299-.929-1.329L3.076 1.56h3.182l5.965 8.532.929 1.329 7.754 11.09h-3.182z"/></svg>
+              <!-- todo(maximsmol): show the handle as text? -->
```

Comment:
I would vote to NOT show the handle, especially since a lot of these accounts are low activity anyway

Reply:
I somewhat disagree. If it is low activity, then it is still useful for archival purposes, especially a historically active chapter that has wealth of resources and can give insights to someone looking to reactivate, but no strong opinions. We can also hide inactive chapters altogether, at the end of the day, semi-activate are easier to reactivated while dead-chapters are similar to non-existing chapters